traffic_portal/app/src/common/modules/form/deliveryServiceJob/form.deliveryServiceJob.tpl.html

```diff
@@ -31,7 +31,12 @@
         <div class="helptext">
             <aside class="warning">
                 <h6>Warning</h6>
-                <p>Delivery service origin(s) must support IMS in order to safely invalidate content</p>
+                <p>REFRESH (preferred default) will mark the matching asset(s) as STALE and an IMS request of the asset(s) from the origin will be performed. Origins that fully adhere 
```

Comment:
I don't think "stale" and "miss" are being used as initialisms here, why are they in all caps?

Reply:
maybe they need to be `stale` and `miss` or something

Reply:
I would probably do `"stale"` and `"miss"` to indicate that a domain-specific meaning of a word somewhat different than its general definition is being used.

Reply:
Done